### PR TITLE
Kriselda/api payment routes

### PR DESCRIPTION
**Pull request by kbonifacio**
Added routes for successful payment and canceled (error) payment. All products have been entered into the stripe UI at this point in time.

### CODE REVIEW

File: routes/payments.js
Comment (Felix-Cadiz) [inline routes/payments.js:8]: This is where we need to add our stripe key. You will need to make an account to receive one.

**Review by Felix-Cadiz — COMMENTED** [review]
Stripe key code needed before approval.

File: routes/payments.js
Comment (kbonifacio) [reply routes/payments.js:8]: So after I made the account, I found that the public stripe key was the same as what Rachel had.

**Review by Felix-Cadiz — APPROVED** [review]

**Pull request merged**
